Casks/l/labplot.rb

```diff
@@ -0,0 +1,27 @@
+cask "labplot" do
+  arch arm: "arm64", intel: "x86_64"
+
+  version "2.11.1"
+  sha256 arm:   "61af87d94f83f9e254b2ada3ae801c9fdc9c034047c13aa7dd10dfc24b846380",
+         intel: "59180a91db2661dc1f0bd52a684213b8ea8f449be3fe108636f96e1438411650"
+
+  url "https://download.kde.org/stable/labplot/labplot-#{version}-#{arch}.dmg"
+  name "LabPlot"
+  desc "Data Visualization and Analysis software"
```

Comment:
```suggestion
  desc "Data visualisation and analysis software"
```

Just a minor nit here to adhere to our style guidelines.

Reply:
Good catch, sorry about that. Should be fixed now.